courses/machine_learning/asl2/recommendation_systems/als_bqml.ipynb

```diff
@@ -0,0 +1,1550 @@
+{
```

Comment:
<p>Typo: Lens</p><p> </p><p>Add attribution to O'Reilly book here</p><p> </p><p>Can you experiment a bit with how to add a link to "deploy this notebook on DLVM" (AI Hub does this <a href="https://cloud.google.com/ai-hub/docs/open-notebook#deploy" target="_blank">https://cloud.google.com/ai-hub/docs/open-notebook#deploy</a>). This is not necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/517/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hmm, gave it the good ol' college try, but it looks like AI hub has something built in to the GCP console. Maybe we can host our notebooks in AI hub as well?